qucs/qucs_init.cpp

```diff
@@ -660,12 +660,12 @@ void QucsApp::initActions()
   helpGetStart->setWhatsThis(tr("Getting Started\n\nShort introduction into Qucs"));
   connect(helpGetStart, SIGNAL(triggered()), SLOT(slotGettingStarted()));
 
-  helpAboutApp = new QAction(tr("&About Qucs-S..."), this);
```

Comment:
The application name should start with capital letter.

Reply:
pr is closed now, this naming doesn't affect macos but affect others. thanks

Reply:
I will correct this myself in the next patch.